tools/common_lib/src/mha.h

```diff
@@ -245,7 +245,7 @@ class MhaBaseDispatcher: public NodeDispatcher
         const auto tensor_input_bytes_width = input_data_.size();
       
         const auto out_shape = get_shape_output();
-        const auto tensor_out_bytes_width = get_tensor_elements_count(out_shape, params_.layout) * get_data_type_bytes_width(params_.dt);
+        const auto tensor_out_bytes_width = get_tensor_elements_count(out_shape, DataLayout::eNHW) * get_data_type_bytes_width(params_.dt);
```

Comment:
This layout is fixed because it might not be the one parsed from command line. Could you also add a output layout entry in the command line for the flexibility?

Reply:
makes sense, will add it.